store/etcd/etcd.go

```diff
@@ -470,6 +567,32 @@ func (s *Etcd) List(directory string) ([]*store.KVPair, error) {
 	return kv, nil
 }
 
+// ListExt lists child nodes of a given directory, etcd native extension.
+func (s *Etcd) ListExt(directory string) (*store.KVPairExt, error) {
```

Comment:
its nothing to do with etcd, its about libkv, above if list returns a single pair, then it matches the behavior of GetExt, so either we need to change the behavior or remove it, since it is redundant.